src/printapp/static/about.html

```diff
@@ -0,0 +1,85 @@
+ <!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="utf-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>About Calvin Web Print</title>
+    <link rel="stylesheet" href="static/css/foundation.min.css">
+    <link rel="stylesheet" href="static/css/normalize.css">
+    <link rel="stylesheet" href="static/css/style.css">
+    <link rel="shortcut icon" href="static/img/favicon.ico" type="image/x-icon">
+    <link rel="icon" href="static/img/favicon.ico" type="image/x-icon">
+</head>
+
+<body>
+  <nav class="top-bar" data-topbar role="navigation">
+    <ul class="title-area">
+      <li class="name">
+        <h1><a href="/">Calvin Web Print</a></h1>
+      </li>
+    </ul>
+  </nav>
+
+  <div class="row">
+    <div class="small-12 columns">
+        <h3 class="subheader">About</h3>
+        <p>
+          Calvin Web Print is the result of a semester-long student project. The goal was to make a friendlier interface to the printing system at Calvin College. The website went live on December 10, 2015, and since then we have attracted many users, received a lot of positive feedback from students, and even got a <a href="http://www.calvin.edu/chimes/2015/03/05/student-website-simplifies-printing-process-for-students/">write-up</a> in Calvin's student newspaper Chimes. The website is currently activley maintained by James Lamine and Tyler VanZanten.
```

Comment:
Once we have a `/stats` page, we can make the "many users" part of "since then we have attracted many users" link to it.